Makefile

```diff
@@ -34,7 +41,7 @@ fix-go:  # auto-fixes all Go lint issues
 	gofmt -s -w ./src ./test
 
 fix-md:  # auto-fixes all Markdown lint issues
-	tools/prettier/node_modules/.bin/prettier --write .
+	tools$/prettier$/node_modules$/.bin$/prettier --write .
```

Comment:
Woah this is weird to me. Why exactly is this needed? These paths don't work on windows?

Reply:
Yes, this is for Windows compatibility. Windows requires backslashes as separators in paths. I agree it's weird but I can't think of a better alternative.